### PR TITLE
Fix gene variants page timeout issue

### PR DESCRIPTION
**Pull request by northwestwitch**
Install on cg-vm1 and go to this page for an institute.

**How to test**:
1. Make sure the page doesn't hang
2. Make sure the HGNC symbol is required to run searches
3. Make sure searches work

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR, DN

### CODE REVIEW

File: scout/server/blueprints/institutes/views.py [hunk 1]
```diff
@@ -11,7 +10,7 @@
 from scout.constants import ACMG_COMPLETE_MAP, ACMG_MAP, CASEDATA_HEADER, CLINVAR_HEADER
 from scout.server.blueprints.variants.controllers import update_form_hgnc_symbols
 from scout.server.extensions import loqusdb, store
-from scout.server.utils import institute_and_case, jsonconverter, templated, user_institutes
+from scout.server.utils import institute_and_case, jsonconverter, templated
```
Comment (dnil) [inline scout/server/blueprints/institutes/views.py:13]: 🐦 🌵
Nice!

File: scout/server/blueprints/institutes/templates/overview/gene_variants.html [hunk 2]
```diff
@@ -86,7 +86,7 @@
 {% macro cell_rank(variant) %}
     <a
        href="{{ url_for('variant.variant', institute_id=variant.institute,
-            case_name=variant.case_display_name, variant_id=variant._id) }}">
+            case_name=variant.case_display_name, variant_id=variant._id) }}" target="_blank">
```
Comment (northwestwitch) [inline scout/server/blueprints/institutes/templates/overview/gene_variants.html:89]: Open variant in a new page

File: scout/adapter/mongo/query.py [hunk 3]
```diff
@@ -151,7 +150,9 @@ def build_variant_query(
             else:
                 LOG.debug("Case %s not found.", similar_case_display_name)
 
-        if select_cases:
+        if (
+            select_cases is not None
```
Comment (dnil) [inline scout/adapter/mongo/query.py:154]: 👍

**Review by dnil — APPROVED** [review]

**Comment by northwestwitch** [comment]
I think I'm going to merge this PR for now since it's doing what is supposed to do. We can address the gene_variants query extension to include the collaborator institutes in another one!

**Pull request merged**